docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
Aaah, interesting. Not sure which is best, my thought was that this was more practical. @jcorbin @Raynos what do you guys think?